specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,402 @@
+          "description": "Title for the potential outage."
+        },
+        "summary": {
+          "type": "string",
+          "description": "Summary for the potential outage."
+        },
+        "confidence": {
+          "type": "number",
+          "description": "Confidence score for the potential outage."
+        },
+        "displayConfidence": {
+          "type": "string",
+          "description": "Display confidence score for the potential outage."
+        },
+        "plane": {
+          "type": "string",
+          "description": "Type of the potential outage: control plane or data plane."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the potential outage."
+        },
+        "correlatedIncidentCount": {
+          "type": "integer",
+          "description": "Number of correlated incidents in the potential outage.",
+          "format": "int64"
+        },
+        "firstReportTime": {
+          "type": "string",
+          "description": "Time and date the potential outage started to become a potential outage."
+        },
+        "lastReportTime": {
+          "type": "string",
+          "description": "Time and date the potential outage was created."
+        },
+        "rawIds": {
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "description": "Ids of child raw potential outages."
+        },
+        "duplicateIds": {
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "description": "Duplicate consolidated outage id."
+        },
+        "outageDeclared": {
+          "type": "boolean",
+          "description": "A value indicating whether there is outage declared for this potential outage."
+        },
+        "expired": {
+          "type": "boolean",
+          "description": "A value indicating whether the potential outage is expired."
+        },
+        "incidents": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/icmIncidents"
+          },
+          "description": "A list of outage incidents."
+        },
+        "isSynthetic": {
+          "type": "boolean",
+          "description": "a value indicating whether the entity is real."
+        }
+      }
+    },
+    "icmIncidents": {
+      "type": "object",
+      "description": "ICM incidents that are related to a potential outage detected by Brain.",
+      "properties": {
+        "id": {
+          "type": "integer",
+          "description": "Id for the incident.",
+          "format": "int64"
+        },
+        "severity": {
+          "type": "integer",
+          "description": "Incident severity.",
+          "format": "int32"
+        },
+        "state": {
+          "type": "string",
+          "description": "Incident state."
+        },
+        "createdDate": {
```

Comment:
>createdDate [](start = 9, length = 11)

Please annotate all the date fields with format: date or format: date-time.

Reply:
addressed